app/views/photos/index.html.erb

```diff
@@ -29,24 +29,26 @@
 
                 <% if photo.comments.any?  %>
                   comments: <br>
-                  <% photo.comments[0..2].each do |comment| %> <br>
-                  <%= comment.user.display_name %> said:<br>
-                  <%= comment.body %><br>
-                  <% if comment_belongs_to_user?(comment) %>
-                    <%= form_for([photo,comment], method: :delete) do |comment| %>
-                      <%= comment.submit "Delete Comment" %><br>
+                  <ul>
```

Comment:
This comments list is reused else where and can be broken up into separate views